packages/dd-trace/src/exporters/common/agent-info-exporter.js

```diff
@@ -68,18 +67,15 @@ class AgentInfoExporter {
     return this._traceBuffer
   }
 
-  resetUncodedTraces () {
-    this._traceBuffer = []
+  exportUncodedTraces () {
+    this.getUncodedTraces().forEach(uncodedTrace => {
+      this.export(uncodedTrace)
+    })
+    this.resetUncodedTraces()
   }
 
-  setUrl (url) {
```

Comment:
Looks like this functionality moved to [packages/dd-trace/src/ci-visibility/exporters/ci-visibility-exporter.js](https://github.com/DataDog/dd-trace-js/pull/2584/files#diff-0d8a13120c2b85f6b44e94a799244a92713104cbe452662442d24c5e4513c598)

Reply:
correct